.gitignore

```diff
@@ -4,6 +4,7 @@ build
 !/modules/oe_theme_helper/js/build
 npm-debug.log
 grumphp.yml
+junit-export
```

Comment:
Is this needed?
I suppose it would only be needed if these files get generated in a local installation.
For me, in oe_bootstrap_theme, I never saw this directory being created. Maybe I need to run phpunit in a specific way?

Reply:
Only now